lib/core/msdf-sys/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "basegl-core-msdf-sys"
+version = "0.1.0"
+authors = ["Adam Obuchowicz <adam.obuchowicz@luna-lang.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+wasm-bindgen = "0.2.53"
+js-sys = "0.3.30"
```

Comment:
Why do you need `js-sys` for this binding? Maybe its necessary, but itsn ot clear to me yet why.

Reply:
I used Uint8Array for passing fonts in memory to js: https://rustwasm.github.io/wasm-bindgen/api/js_sys/struct.Uint8Array.html